# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/layers/ipc/LayersMessageUtils.h
# Commit: f37913e7cb92
# Full Hash: f37913e7cb9295617b7e5a7ac5f462f29df583b4
# Author: Matt Woodrow <mwoodrow@mozilla.com>
# Date: 2021-08-18 15:53:06
# Regressor Bug: 1726063
# File Overlap Count: 1
# Description:
#   Bug 1726063 - Remove support for non-WR compositor initialization. r=jrmuizel,aosmond
#   
#   Depends on D122796
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D122798
# ==============================================================================

diff -r 7398bc391379 -r f37913e7cb92 gfx/layers/ipc/LayersMessageUtils.h
--- a/gfx/layers/ipc/LayersMessageUtils.h	Tue Aug 17 23:19:44 2021 +0000
+++ b/gfx/layers/ipc/LayersMessageUtils.h	Tue Aug 17 23:19:44 2021 +0000
@@ -863,7 +863,6 @@
 
   static void Write(Message* aMsg, const paramType& aParam) {
     WriteParam(aMsg, aParam.mUseAPZ);
-    WriteParam(aMsg, aParam.mUseWebRender);
     WriteParam(aMsg, aParam.mUseSoftwareWebRender);
     WriteParam(aMsg, aParam.mAllowSoftwareWebRenderD3D11);
     WriteParam(aMsg, aParam.mAllowSoftwareWebRenderOGL);
@@ -874,7 +873,6 @@
   static bool Read(const Message* aMsg, PickleIterator* aIter,
                    paramType* aResult) {
     return ReadParam(aMsg, aIter, &aResult->mUseAPZ) &&
-           ReadParam(aMsg, aIter, &aResult->mUseWebRender) &&
            ReadParam(aMsg, aIter, &aResult->mUseSoftwareWebRender) &&
            ReadParam(aMsg, aIter, &aResult->mAllowSoftwareWebRenderD3D11) &&
            ReadParam(aMsg, aIter, &aResult->mAllowSoftwareWebRenderOGL) &&